diagnostic_rt_qpcr/operation_types/run_qpcr/protocol.rb

```diff
@@ -81,6 +87,27 @@ def main
 
     {}
   end
+  
+    def flick_to_remove_bubbles(paired_ops)
+      show do
+        title 'Examin for Bubbles'
+        note 'Examin all wells in plates for bubbles'
+        note 'If there are bubles gently flick plate to until bubbles are gone'
```

Comment:
Spelling. Also, I'm not sure how to flick a plate. It could easily make a mess of things.